deploy/image/csi-image-plugin.yaml

```diff
@@ -0,0 +1,90 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-imageplugin
+spec:
+  selector:
+    matchLabels:
+      app: csi-imageplugin
+  template:
+    metadata:
+      labels:
+        app: csi-imageplugin
+    spec:
+      hostNetwork: true
+      containers:
+        - name: node-driver-registrar
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.0.2
+          imagePullPolicy: Always
+          lifecycle:
+            preStop:
+              exec:
+                command: ["/bin/sh", "-c", "rm -rf /registration/csi-image /registration/csi-image-reg.sock"]
+          args:
+            - --v=5
+            - --csi-address=/csi/csi.sock
+            - --kubelet-registration-path=/var/lib/kubelet/plugins/csi-image/csi.sock
+          env:
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+          volumeMounts:
+          - mountPath: /csi
+            name: socket-dir
+          - mountPath: /registration
+            name: registration-dir
+
+        - name: image
+          image: quay.io/k8scsi/imageplugin:v1.0.1
```

Comment:
Is this a valid version? should it be canary for now?

Reply:
+1

What should the actual repo be too?

Reply:
I think the repo is fine. Maybe just canary as the tag